### PR TITLE
LPD-33056 Applying UTC timeZone for all-day calendar events

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://liferay.atlassian.net/browse/LPD-33056

### CODE REVIEW

**Review by cgmonte — COMMENTED** [review]
Hey, @carolmariaabb, looks almost perfect to me! The only thing that I would like to point is that `.locator()` is not the optimal way of selecting elements, because if anything changes in the DOM in the future, does locators will stop working. For every call of `.locator()`, please check if there is a [Locator](https://playwright.dev/docs/locators) that would work. If you have done this already, please let me kown and re-request this review. Locators (with capital L) does not rely on specific DOM attributes and are more resilient to changes in the DOM structure.

**Comment by carolmariaabb** [comment]
ci:test:sf

File: modules/test/playwright/pages/calendar-web/CalendarWidgetPage.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		this.addEventButton = page.getByRole('button', {name: 'Add Event'});
+		this.allDayCheckbox = page
+			.frameLocator('iframe')
+			.getByRole('checkbox', {
+				exact: true,
+				name: 'All Day',
+			});
+		this.calendarWidget = page.locator(
+			'.lfr-layout-structure-item-com-liferay-calendar-web-portlet-calendarportlet'
+		);
```
Comment (carolmariaabb) [inline modules/test/playwright/pages/calendar-web/CalendarWidgetPage.ts:31]: @cgmonte as discussed, I wasn't able to get rid of this locator

Comment (cgmonte) [reply modules/test/playwright/pages/calendar-web/CalendarWidgetPage.ts:31]: That's OK! Everytime I review a Playwright PR, I ask if the author tried to replace locators with Locators. I just want people to try. Many times it is possible to find a working Locator in few minutes. If this is not the case, just go for the regular locator, which you did here. Thanks

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:playwright

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**